README.md

```diff
@@ -50,20 +50,27 @@ go install github.com/mineiros-io/terramate/cmd/terramate@<version>
 ```
 
 Where **<version>** is any terramate [version tag](https://github.com/mineiros-io/terramate/tags),
-or you can just install the **latest** using go install:
+or you can just install the **latest** release:
 
 ```
 go install github.com/mineiros-io/terramate/cmd/terramate@latest
 ```
 
+To install the latest code on main:
+
+```
+export GOPROXY=direct
+go install github.com/mineiros-io/terramate/cmd/terramate@main
+```
+
 Or if you have the project cloned locally just run:
 
 ```
 make install
 ```
 
 We put great effort into keeping the main branch stable, so it should be safe
-to use **latest** to play around, but not recommended for long term automation
+to use **main** to play around, but not recommended for long term automation
```

Comment:
isn't just exporting GOPROXY enough?

Reply:
Enough for what ? =P

Reply:
enough to solve the cache issue :laughing:

Reply:
ahhhh, I think so. From my local tests it didn't seemed related to caching at all, I deleted all the cache + all local versions of modules and yet it was downloading an older version x_x.